python/modules/BTagSelection.py

```diff
@@ -19,42 +19,25 @@ class BTagSelection(Module):
     
     def __init__(
         self,
+        btaggingWP = {},
         inputCollection=lambda event: Collection(event, "Jet"),
-        flagName = "isBTagged",
-        outputName="btaggedJets",
+        outputName_list= [], #"btaggedJets",
         jetMinPt=30.,
         jetMaxEta=2.4,
-        workingpoint = TIGHT,
-        storeKinematics=['pt', 'eta'],
+        workingpoint = [], 
```

Comment:
same as for the lepton selection; if one likes multiple workinpoints it might be best to have multiple modules